frontend/packages/network-attachment-definition-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as _ from 'lodash';
+import {
+  NetworkAttachmentDefinitionConfig,
+  NetworkAttachmentDefinitionKind,
+} from '../components/network-attachment-definitions/types';
+
+export const getConfigAsJSON = (
+  obj: NetworkAttachmentDefinitionKind,
+): NetworkAttachmentDefinitionConfig => {
+  try {
+    return JSON.parse(_.get(obj, 'spec.config'));
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error('Unable to parse network attachment definition configuration');
+    return null;
+  }
+};
+
+export const getType = (config: NetworkAttachmentDefinitionConfig): string => {
```

Comment:
we are starting to use a new type of selectors 
```tsx
_.get(config, 'type') === undefined ? config.type : null
```

Reply:
null should be probably a defaultValue

Reply:
Done!